src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -105,24 +105,24 @@ void printJournalFileMeta(bsl::ostream&                    ostream,
                           const mqbs::JournalFileIterator* journalFile_p,
                           bslma::Allocator*                allocator)
 {
-    BSLS_ASSERT_SAFE(journalFile_p && journalFile_p->isValid());
+    if (!journalFile_p || !journalFile_p->isValid()) {
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.